src/test/java/org/jilt/test/data/annotations/NonParameterAnnotation.java

```diff
@@ -0,0 +1,9 @@
+package org.jilt.test.data.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+@Target({ElementType.TYPE, ElementType.FIELD, ElementType.METHOD, ElementType.CONSTRUCTOR, ElementType.LOCAL_VARIABLE,
+        ElementType.ANNOTATION_TYPE, ElementType.PACKAGE, ElementType.TYPE_PARAMETER})
```

Comment:
I did not include `MODULE` and `RECORD_COMPONENT` to prevent compatibility checks from failing

Reply:
Like I said above, this is immaterial. Change it to have just `ElementType.FIELD`.